.ci/validation.sh

```diff
@@ -594,9 +594,9 @@ javac19)
   fi
   ;;
 
-jdk14-assembly-site)
+packageAssembly-siteNolinkcheck)
```

Comment:
All case statement patterns should be in kebab case, please do `package-assembly-site` or something similar. "No link checks" is just details that users do not need to know.